qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -2819,6 +2841,22 @@ void VulkanPipelineStateViewer::setState()
 
       if(showNode(usedSlot, /*filledSlot*/ true))
       {
+        QString writemask = QFormatStr("%1%2%3%4")
+                                .arg((blend.writeMask & 0x1) == 0 ? lit("_") : lit("R"))
+                                .arg((blend.writeMask & 0x2) == 0 ? lit("_") : lit("G"))
+                                .arg((blend.writeMask & 0x4) == 0 ? lit("_") : lit("B"))
+                                .arg((blend.writeMask & 0x8) == 0 ? lit("_") : lit("A"));
+
+        // With VK_KHR_dynamic_rendering_local_read, if a color attachment is mapped to
+        // VK_ATTACHMENT_UNUSED, it is implicitly disabled.  The Slot name in the "Render Pass"
+        // pane already tags the attachment with [disabled], so the write mask is simply set to
+        // ____ here.
+        if(i < rp.colorAttachmentLocations.count() &&
+           rp.colorAttachmentLocations[i] == VK_ATTACHMENT_UNUSED)
+        {
+          writemask = lit("____");
```

Comment:
I think we should have a different display here so there's no chance of confusion about why writes are dropped. It's not really the write mask at all but I can appreciate the logic of putting something here. Setting it to `DISABLED` or `UNUSED` would be more clear I feel.